### PR TITLE
feature/FOUR-13852: Launchpad Setup - Create a modal for Launch Settings launchpad

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
Create a modal for Launch Settings launchpad
The change related to the ModalSaveVersion will updated from Engines Team

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-13852

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

### CODE REVIEW

File: resources/js/components/shared/LaunchpadSettingsModal.vue [hunk 1]
```diff
@@ -0,0 +1,759 @@
...
+      const params = {
+        indexImage: index,
+        type: "delete",
+      };
+      ProcessMaker.EventBus.$emit("getLaunchpadImagesEvent", params);
+    },
+    /**
+     * Method to store version info from Launchpad Window
+     */
+    saveFromEditLaunchpad() {
```
Comment (fagubla) [inline resources/js/components/shared/LaunchpadSettingsModal.vue:385]: this is not necessary

File: resources/js/components/shared/LaunchpadSettingsModal.vue [hunk 2]
```diff
@@ -0,0 +1,759 @@
...
+          this.types[this.options.type],
+          this.redirectUrl,
+          this.nodeId,
+          this.options.type === "Screen" ? (false, resolve) : resolve,
+          reject,
+          this.types[this.options.type] === "modeler-save" ? false : null,
+        );
+      });
+
+      promise
```
Comment (fagubla) [inline resources/js/components/shared/LaunchpadSettingsModal.vue:462]: this promise make change in the version, it's not necessary

File: resources/js/components/shared/LaunchpadSettingsModal.vue [hunk 3]
```diff
@@ -0,0 +1,759 @@
...
+        }
+      } else {
+        this.processDescriptionInitial = this.descriptionSettings;
+      }
+    },
+    /**
+     * Method to retrieve data from process description field
+     */
+    getProcessDescription() {
+      if (this.origin !== "core") {
```
Comment (fagubla) [inline resources/js/components/shared/LaunchpadSettingsModal.vue:541]: review why is doing this if statement? and if is necessary origin

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/components/shared/LaunchpadSettingsModal.vue [hunk 4]
```diff
@@ -0,0 +1,687 @@
...
+    },
+    launchpadIconSelected(iconData) {
+      this.selectedLaunchpadIcon = iconData.value;
+      this.selectedLaunchpadIconLabel = iconData.label;
+    },
+  },
+};
+</script>
+
+<style lang="scss">
```
Comment (pmPaulis) [inline resources/js/components/shared/LaunchpadSettingsModal.vue:593]: please add scoped

File: resources/js/components/shared/LaunchpadSettingsModal.vue [hunk 5]
```diff
@@ -0,0 +1,687 @@
...
+            id="statusDropdown"
+            class="btn dropdown-toggle dropdown-style w-100 d-flex justify-content-between align-items-center btn-custom"
+            type="button"
+            data-toggle="dropdown"
+            aria-haspopup="true"
+            aria-expanded="false"
+          >
+            <div class="d-flex align-items-center">
+              <i class="far fa-chart-bar" />
+              <span class="ml-2 custom-text">{{ selectedSavedChart || 'Select Chart' }}</span>
```
Comment (pmPaulis) [inline resources/js/components/shared/LaunchpadSettingsModal.vue:142]: Add the translation for `'Select Chart'`

File: resources/js/components/shared/LaunchpadSettingsModal.vue [hunk 6]
```diff
@@ -0,0 +1,687 @@
...
+            aria-labelledby="statusDropdown"
+          >
+            <a
+              v-for="(item, index) in dropdownSavedCharts"
+              :key="index"
+              class="dropdown-item"
+              @click="selectOption(item)"
+            >
+              <i class="far fa-chart-bar custom-text" />
+              {{ item.title || 'Select Chart' }}
```
Comment (pmPaulis) [inline resources/js/components/shared/LaunchpadSettingsModal.vue:156]: Add the translation for 'Select Chart'

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Pull request closed without merging**